### DISCUSSION TITLE
en/2020/02/25/b2b-communication/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#167] [Announcements]
# en/2020/02/25/b2b-communication/

https://hongtaoh.com/en/2020/02/25/b2b-communication/

**Comment by daoyuanw** [comment]
有过类似的想法，我想写的是通过监测梦境来搭建Dream Net，看完很受启发，比如感官欢愉成瘾这一点之前从来没思考过。
